src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -2028,4 +2028,100 @@ BOOST_AUTO_TEST_CASE(GradientReparametrizationLayerBetaTest)
   BOOST_REQUIRE_LE(CheckGradient(function), 1e-4);
 }
 
+/**
+ * Simple sequential module residual block test.
+ */
+BOOST_AUTO_TEST_CASE(SimpleSequentialLayerResidualBlockTest)
+{
+  arma::mat outputA /* sequential */, outputB /* residual */, input,
+      deltaA /* sequential */, deltaB /* residual */;
+  Sequential<>* sequential = new Sequential<>(true, false);
+  Sequential<>* residual = new Sequential<>(true, true);
+
+  Linear<>* linearA = new Linear<>(10, 10);
+  linearA->Parameters().randu();
+  linearA->Reset();
+  Linear<>* linearB = new Linear<>(10, 10);
+  linearB->Parameters().randu();
+  linearB->Reset();
+
+  // Add the same layers(with the same parameters) to both normal Sequential and
+  // residual Sequential object.
```

Comment:
Please add a space after `layers`. Also please change `normal Sequential and residual Sequential object` to simply `normal and residual object`.

Reply:
I kept the names because naming them `normal` and `residual` objects will be confusing as there are no actual `Normal` and `Residual` objects in our API. I do agree that the `normal` is unnecessary, so I will remove it and keep the rest. I hope that's okay.